packages/slice-machine/test/core/libraries/screenshot.test.ts

```diff
@@ -1,57 +0,0 @@
-import { afterEach, expect, test, vi } from "vitest";
```

Comment:
The tests in this file are no longer relevant since Slice Machine UI doesn't handle writing and reading screenshots.

Tests for screenshots can be found in `@slicemachine/manager`. The new tests aren't a 1-to-1 replacement for the deleted tests, but they cover the new flow from start the finish. For example:

- Capturing Slice screenshots: https://github.com/prismicio/slice-machine/blob/dev-plugins/packages/manager/test/SliceMachineManager-screenshots-captureSliceSimulatorScreenshot.test.ts
- Uploading screenshots: https://github.com/prismicio/slice-machine/blob/dev-plugins/packages/manager/test/SliceMachineManager-screenshots-uploadScreenshot.test.ts
- Reading Slice screenshots: https://github.com/prismicio/slice-machine/blob/dev-plugins/packages/manager/test/SliceMachineManager-slices-readSliceScreenshot.test.ts
- Updating/writing Slice screenshots: https://github.com/prismicio/slice-machine/blob/dev-plugins/packages/manager/test/SliceMachineManager-slices-updateSliceScreenshot.test.ts